tools/pfe-tools/dev-server/plugins/pfe-dev-server.ts

```diff
@@ -130,6 +130,11 @@ function getRouter(options: PfeDevServerInternalConfig) {
       return next();
     })
 
+    // Redirect `core/pfe-core/controllers/thingy.js` to `core/pfe-core/controllers/thingy.ts` for requests not previously handled
```

Comment:
is there anything we should be worried about downstream here?  eg `lib/elements` in RHDS?

Reply:
follow up, should we allow for optional routes?

Reply:
downstream can do their own middleware i think

Reply:
You are correct and we are already doing as such:

```
export default pfeDevServerConfig({
  litcssOptions,
  tsconfig: 'tsconfig.json',
  middleware: [
    /** redirect requests for lightdom css to /elements */
    function(ctx, next) {
      const match = ctx.path.match(/^\/components\/(?<slug>[-\w]+)\/(?<path>.*)\.css$/);
      if (match) {
        const { slug, path } = /** @type{{ slug: string; path: string }} */ (match.groups);
        ctx.redirect(`/elements/rh-${slug}/${path}.css`);
      }
      return next();
    },
    /** redirect requests for /assets/* css to /docs/assets/prism.css */
    function(ctx, next) {
      if (ctx.path.startsWith('/assets/')) {
        ctx.redirect(`/docs${ctx.path}`);
      }
      return next();
    }
  ]
});
```